modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplateFilesTest.java

```diff
@@ -338,6 +338,12 @@ private void _testArchetypeMetadataXml(
 		Path archetypePomXmlPath = projectTemplateDirPath.resolve(
 			"src/main/resources/archetype-resources/pom.xml");
 
+		String pathString = archetypePomXmlPath.toString();
+
+		if (pathString.contains("ext")) {
```

Comment:
Is this code OK to filter out the ext template project that it not support maven?